packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -6,7 +6,6 @@ import {IRuntimeEnvironment} from 'coveo.analytics';
 export const localeValidation = new StringValue({
   emptyAllowed: false,
   required: false,
-  regex: /[a-z]{2}-[A-Z]{2}/,
```

Comment:
I can't remember what exactly I was thinking when I set that regex validation on `locale`, but it's not valid/complete.

I could re-introduce one that accept things like `fr` AND `fr-CA`, but... at the end of the day, I'm not even sure it brings any "value" for the validation aspect.

It does not tell actually tell you what is the valid list of all iso code that exists.

Open to opinion if you prefer me to re-add a more complex regex (which means the validation message will be ... more cryptic than it was before), or to leave it like that as "any string will do".

Reply:
I'd be for simplifying 👍